packages/xo-web/src/xo-app/home/index.js

```diff
@@ -649,12 +661,26 @@ export default class Home extends Component {
     }
 
     const sort = this._getDefaultSort(props)
+    const {
+      $container: selectedHosts,
+      $pool: selectedPools,
+      power_state: powerStates,
+      resourceSet,
+      tags,
+    } = properties
 
     this.setState({
-      selectedHosts: properties.$container,
-      selectedPools: properties.$pool,
-      selectedTags: properties.tags,
-      selectedResourceSets: properties.resourceSet,
+      selectedHosts,
+      selectedPools,
+      selectedPowerStates:
+        powerStates === undefined
+          ? []
+          : (props.type === 'VM'
+              ? POWER_STATE_VM
+              : POWER_STATE_HOST
+            ).filter(option => powerStates.includes(option.value)),
```

Comment:
You should be able to just pass `powerStates`. (Maybe with a `.toLowerCase()` too?)